src/style/price.css

```diff
@@ -0,0 +1,23 @@
+.product__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  align-items: center;
+}
+  
+.product__price-text {
+  font-family: "Roboto", sans-serif;
```

Comment:
move font styles to html selector and change font-size and font-weight only in some places

Reply:
i did